From db10fb1853602b91d1b99bab3fa0fdf7c22b1af1 Mon Sep 17 00:00:00 2001
From: b02247 <b02247@freescale.com>
Date: Fri, 11 Jan 2013 17:34:51 +0800
Subject: [PATCH 147/310] ENGR00235540 remove ro property for device partition

The "/device" always is empty. When use ubifs, it will forbit
read only & empty volume, mount will fail.

Signed-off-by: b02247 <b02247@freescale.com>
---
 imx6/etc/fstab.freescale | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/imx6/etc/fstab.freescale b/imx6/etc/fstab.freescale
index 48d3ab2..aed23a5 100644
--- a/imx6/etc/fstab.freescale
+++ b/imx6/etc/fstab.freescale
@@ -13,4 +13,4 @@
 #ubi0:system	/system		ubifs 	ro					wait
 #ubi0:data       /data           ubifs   nosuid,nodev,nodiratime,noatime         wait
 #ubi0:cache      /cache          ubifs   nosuid,nodev                            wait
-#ubi0:device     /device         ubifs   ro,nosuid,nodev                         wait
+#ubi0:device     /device         ubifs   nosuid,nodev                         wait
-- 
1.8.0

